misc/editor/rustrover/run-environmentd.run.xml

```diff
@@ -0,0 +1,30 @@
+<!--
+Copyright Materialize, Inc. and contributors. All rights reserved.
+
+Use of this software is governed by the Business Source License
+included in the LICENSE file at the root of this repository.
+
+As of the Change Date specified in that file, in accordance with
+the Business Source License, use of this software will be governed
+by the Apache License, Version 2.0.
+-->
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="run-environmentd" type="CargoCommandRunConfiguration" factoryName="Cargo Command">
+    <option name="buildProfileId" value="dev" />
+    <option name="command" value="run --bin environmentd -- --internal-http-listen-addr=0.0.0.0:6878 --orchestrator=process --orchestrator-process-secrets-directory=$PROJECT_DIR$/mzdata/secrets --orchestrator-process-tcp-proxy-listen-addr=0.0.0.0 --orchestrator-process-prometheus-service-discovery-directory=$PROJECT_DIR$/mzdata/prometheus --orchestrator-process-scratch-directory=$PROJECT_DIR$/scratch --secrets-controller=local-file --persist-consensus-url=postgres://root@localhost:26257/materialize?options=--search_path=consensus --persist-blob-url=file://$PROJECT_DIR$/mzdata/persist/blob --timestamp-oracle-url=postgres://root@localhost:26257/materialize?options=--search_path=tsoracle --environment-id=local-az1-d205081d-4387-4daa-9f29-b70b1fc496e0-0 --bootstrap-role=materialize" />
```

Comment:
This seems hard to maintain. As long as the blast radius of this approach is limited to misc/editor/rustrover that seems fine—you'll just have to update this as necessary for your workflow as run.py evolves.

An alternative to consider is adding a `--skip-build` option to bin/environmentd. That _is_ a bit of extra complexity for run.py, but it's complexity that I'd be willing to absorb to make these editor-specific configurations more maintainable. The `--skip-build` option would be mutually exclusive with the `--build-only` option and when specified would skip the Cargo/Bazel build step and instead expect the binaries to be already built. Then you'd just have to figure out how to adjust this run configuration to build the binaries, put them in the spot that bin/environmentd expects, and then invoke `bin/environmentd --skip-build`.

That way you'd automatically pick up new changes to bin/environmentd without needing to update your RustRover configuration.